src/norm/sqlite/sqlgen.nim

```diff
@@ -77,17 +77,17 @@ proc getDbType(fieldRepr: FieldRepr): string =
   result =
     if fieldRepr.typ.kind in {nnkIdent, nnkSym}:
       case $fieldRepr.typ
-        of "int", "Positive", "Natural", "bool", "DateTime": "INTEGER NOT NULL"
-        of "string": "TEXT NOT NULL"
-        of "float": "REAL NOT NULL"
-        else: "TEXT NOT NULL"
+      of "int", "Positive", "Natural", "bool", "DateTime": "INTEGER NOT NULL DEFAULT 0"
+      of "string": "TEXT NOT NULL DEFAULT ''"
+      of "float": "REAL NOT NULL DEFAULT 0"
+      else: "TEXT NOT NULL DEFAULT ''"
     elif fieldRepr.typ.kind == nnkBracketExpr and $fieldRepr.typ[0] == "Option":
       case $fieldRepr.typ[1]
-        of "int", "Positive", "Natural", "bool", "DateTime": "INTEGER"
-        of "string": "TEXT"
-        of "float": "REAL"
-        else: "TEXT"
-    else: "TEXT NOT NULL"
+      of "int", "Positive", "Natural", "bool", "DateTime": "INTEGER "
```

Comment:
Why the trailing space after INTEGER?

Reply:
Also, why change the case formatting? I know it's debatable whether ofs should be indented but this is the way the rest of the case blocks are formatted, so I suggest we conform to it here.